csharp/ql/lib/semmle/code/csharp/Unification.qll

```diff
@@ -545,28 +490,15 @@ module Gvn {
      * type parameters themselves) to make the two substituted terms equal.
      */
     cached
-    predicate unifiable(ConstructedGvnType t1, ConstructedGvnType t2) {
-      unifiableSingle(t1, t2)
-      or
-      exists(CompoundTypeKind k | unifiableMultiple(k, t1, t2, k.getNumberOfTypeParameters() - 1))
-    }
+    predicate unifiable(ConstructedGvnType t1, ConstructedGvnType t2) { unifiable(t1, t2, _) }
 
     /**
      * Holds if GVN `t1` subsumes GVN `t2`. That is, is it possible to replace all
```

Comment:
Unrelated to your changes, there's a typo in this comment. `That is, is it ...` should be `That is, it is ...`

Reply:
Thanks; fixed.